daisy_integration_tests/scripts/post_translate_test.sh

```diff
@@ -37,6 +50,21 @@ function wait_for_connectivity {
   exit 1
 }
 
+# Assert that a service is running; supports upstart and systemd.
```

Comment:
is this satisfactory? we don't need any sysvinit support for any imports? also is the `service` binary available everywhere? that's the legacy sysvinit tool, and on systems which have transitioned to systemd, it becomes a wrapper around systemd. if you're intending to support systemd specifically, you should use `systemctl` instead of `service`

Reply:
It wasn't intended to describe what import supports, but rather my understanding of what the previous code was checking for. Perhaps a more general: "Assert that a service is running; first trying `initctl`, then trying `service`". Or maybe just "Assert that a service is running"

Regarding switching to using `systemctl`, or whether `service` is available everywhere, I'm fine with a pragmatic approach of adding more fallbacks if/when they become necessary (so far these two cover all of the test images we have, which roughly cover all of the supported distro/version combos)